cmd/minishift/cmd/config/config.go

```diff
@@ -77,6 +77,9 @@ var (
 	OpenshiftEnv      = createFlag("openshift-env", nil, nil, nil, false)
 	Metrics           = createFlag("metrics", SetBool, nil, nil, true)
 	Logging           = createFlag("logging", SetBool, nil, nil, true)
+	ServiceCatalog    = createFlag("service-catalog", SetBool, nil, nil, true)
+	// ability to specify additional cluster up flags
+	OpenshiftExtraFlags = createFlag("openshift-extra-flags", SetString, nil, nil, true)
```

Comment:
I had the same thought, but I followed Jorge's suggestion on #1137 (and it seemed Hardy had no comments at the time)